docs/adr/0006_published_projects.md

```diff
@@ -0,0 +1,43 @@
+# [API engineering] Published projects
+
+* Date: 12 Oct 2021
+* Status: draft
+* Deciders: Alicia Arenzana, Kamil Gajowy, Alex Larrañaga, Dominik Ostrowski, Andrea Rota, Maciej Sikorski
+
+## Context and problem statement
+We need to decide how to implement published projects described [here](../hld/projects/publish/high-level-design.md)
+
+## Decision drivers
+
+
+## Considered options
+### Solution 1
+As the projects table already contains the is_public column we can add new columns related to the published version of the project.
+
+#### Pros:
+* Simple to add
+#### Cons:
+* It will increase the complexity of a project
+* It can strongly affect the projectsCrudService
+
+### Solution 2
+We can add a new entity PublishedProject and table which will have its own name, description, ect. and will be in a 0-1 to 1 relation with Project.
+
+#### Pros:
+* This will separate the logic, and CRUD of these two cases.
+#### Cons:
+* We have to do a migration to the new db structure.
```

Comment:
Could be Solution 2 A:

`ProjectCrud` can base on some ProjectView (as true read model) which may connect both entities.

Reply:
Should I put it as it is or you want to add it? TBH I don't know how it can work with the BaseService

Reply:
@kgajowy 
are you sure that if we make `ProjectCrud` to base on a view, we will be able to use it for updates?